modules/gltf/src/gltf/gltf-parser.js

```diff
@@ -206,10 +224,29 @@ export default class GLTFParser {
 
   // PARSING HELPERS
 
+  _getFullUri(uri, base) {
+    if (uri.startsWith('data:') || uri.startsWith('http:') || uri.startsWith('https:')) {
+      return uri;
+    }
+
+    return base.substr(0, base.lastIndexOf('/') + 1) + uri;
+  }
+
   // Start loading linked assets
   _loadLinkedAssets(options) {
-    // TODO: Not implemented
-    // TODO: Return a promise?
+    const promises = [];
+
+    if (!this.glbParser) {
+      this.gltf.buffers.forEach(buffer => {
+        if (buffer.uri) {
+          promises.push(window.fetch(this._getFullUri(buffer.uri, options.uri)).then(res => res.arrayBuffer()).then(data => {
```

Comment:
Maybe use loaders.gl [`readFile`](https://github.com/uber-web/loaders.gl/blob/master/docs/api-reference/core/read-file.md) API so that this works on node.js as well.